static/src/javascripts/projects/common/modules/identity/consents.js

```diff
@@ -27,35 +27,25 @@ const isLoadingClassName = 'loading';
 const optOutClassName = 'fieldset__fields--opt-out';
 const optInClassName = 'fieldset__fields--opt-in';
 
-const requestDebounceTimeout = 150;
-
 const LC_CHECK_ALL = 'Select all';
 const LC_UNCHECK_ALL = 'Deselect all';
 const UNSUBSCRIPTION_SUCCESS_MESSAGE =
     "You've been unsubscribed from all Guardian marketing newsletters and emails.";
 const ERR_MALFORMED_HTML = 'Something went wrong';
 
-const submitPartialConsentFormDebouncedRq: ({}) => Promise<void> = debounce(
-    formData =>
-        reqwest({
-            url: '/privacy/edit-ajax',
-            method: 'POST',
-            data: formData,
-        }),
-    requestDebounceTimeout
-);
-const submitPartialConsentFormData = {};
-
-const submitPartialConsentForm = (formData: {}): Promise<void> => {
-    Object.assign(submitPartialConsentFormData, formData);
-    return submitPartialConsentFormDebouncedRq(
-        submitPartialConsentFormData
-    ).then(() => {
-        Object.keys(submitPartialConsentFormData).forEach(_ => {
-            delete submitPartialConsentFormData[_];
-        });
+const updateConsent = (consent: {
```

Comment:
nitpicky but you can create

```
type Consent = {
        id: string,
        consented: boolean,
}
```

and then use type definition across the board